playground.rs

```diff
@@ -0,0 +1,34 @@
+
+    println!("node_idx: {:?} node_visits: {}", &node_idx, &node_visits);
```

Comment:
Perhaps we might want to have a `scratchpad` folder where we put these kind of files?

Reply:
see above